Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -216,8 +215,8 @@ + (BranchActivityItemProvider *)getBranchActivityItemWithParams:(NSDictionary *)
 
 #pragma mark - Configuration methods
 
-+ (void)setDebug {
-    [BNCPreferenceHelper setDevDebug];
+- (void)setDebug {
```

Comment:
but there are likely hundreds of apps with static method commented out or in some debug logic that will now break the next time they update the SDK files. There's nothing worse than that when you're in a hurry